x-pack/plugins/security_solution/cypress/e2e/detection_alerts/event_rendered_view.cy.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getNewRule } from '../../objects/rule';
+import {
+  EVENT_SUMMARY_ALERT_RENDERER_CONTENT,
+  EVENT_SUMMARY_COLUMN,
+  ALERT_RENDERER_HOST_NAME,
+} from '../../screens/alerts';
+import { FIELDS_BROWSER_VIEW_BUTTON } from '../../screens/fields_browser';
+import {
+  DATA_GRID_COLUMN_ORDER_BTN,
+  DATA_GRID_FIELD_SORT_BTN,
+} from '../../screens/common/data_grid';
+import { HOVER_ACTIONS } from '../../screens/timeline';
+import {
+  switchAlertTableToEventRenderedView,
+  waitForAlerts,
+  waitForTopNHistogramToLoad,
+} from '../../tasks/alerts';
+import { createCustomRuleEnabled } from '../../tasks/api_calls/rules';
+import { cleanKibana } from '../../tasks/common';
+import { login, visit } from '../../tasks/login';
+import { ALERTS_URL } from '../../urls/navigation';
+import { TOP_N_ALERT_HISTOGRAM, TOP_N_CONTAINER_CLOSE_BTN } from '../../screens/shared';
+
+describe('Event Rendered View', { testIsolation: false }, () => {
```

Comment:
I would make a note about how `testIsolation` is disabled for performance reasons, and how it's unnecessary for these tests. My only concern is other people adding tests to this suite not knowing that the tests aren't isolated, but I guess they'll figure it out when they run them 🤷🏾‍♂️